scripts/build.js

```diff
@@ -41,12 +41,15 @@ const JS_FILES_PATTERN = '**/*.js';
 const IGNORE_PATTERN = '**/__{tests,mocks}__/**';
 const PACKAGES_DIR = path.resolve(__dirname, '../packages');
 
-const INLINE_REQUIRE_BLACKLIST = /packages\/expect|(jest-(circus|diff|get-type|jasmine2|matcher-utils|message-util|regex-util|snapshot))|pretty-format\//;
+// const INLINE_REQUIRE_BLACKLIST = /packages\/expect|(jest-(circus|diff|get-type|jasmine2|matcher-utils|message-util|regex-util|snapshot))|pretty-format\//;
 
-const transformOptions = JSON.parse(
```

Comment:
Here, my recommendation would be
```
const transformOptions = {
  cwd: path.resolve(__dirname, ".."),
  // configFile: "babel.config.js" // optional, since the `cwd` is the root and it will autodetect.
  babelrc: false,
};
```